pkg/nsx/services/common/store_test.go

```diff
@@ -177,5 +177,7 @@ func Test_InitializeResourceStore(t *testing.T) {
 		})
 	defer patches2.Reset()
 
-	service.InitializeResourceStore(&wg, fatalErrors, ResourceTypeRule, ruleStore)
+	service.InitializeResourceStore(&wg, fatalErrors, ResourceTypeRule, nil, ruleStore)
+	mTag, mScope := TagScopeNamespace, "11111"
+	service.InitializeResourceStore(&wg, fatalErrors, ResourceTypeRule, []model.Tag{{Tag: &mTag, Scope: &mScope}}, ruleStore)
```

Comment:
Please add verification logic in UT, not only here but any other places as well

Reply:
OK